offerzen/global/employers/prospects@2.1.1.js

```diff
@@ -0,0 +1,154 @@
+window.$loaded(function () {
```

Comment:
The latest prospects file is 2.2. Please implement the payload change on that one instead as there are a lot of differences between 2.1 and 2.2

Reply:
@jethroflanagan prospects 2.2 isn't in use on production yet. From chatting to @chama-ram , there is still some things that need to be done on that, which is why I went and used the existing prospects 2.1 file